new-lamassu-admin/src/pages/Commissions/Commissions.js

```diff
@@ -69,51 +87,66 @@ const Commissions = ({ name: SCREEN_KEY }) => {
     return saveConfig({ variables: { config } })
   }
 
-  const onEditingDefault = (it, editing) => setEditingDefault(editing)
-  const onEditingOverrides = (it, editing) => setEditingOverrides(editing)
+  const saveOverridesFromList = it => (_, override) => {
+    const cryptoOverriden = R.path(['cryptoCurrencies', 0], override)
+
+    const machineOverrides = R.map(removeCoinFromOverride(cryptoOverriden))(
+      R.filter(
+        c =>
+          c.machine === override.machine &&
+          !c.cryptoCurrencies.every(c => c === cryptoOverriden)
```

Comment:
Why do we need this `every` line? From what I'm understanding it's just filtering out the overrides where the machine/crypto pair happens (which should only be the current one)

Reply:
The `every` filters out the cases where the list only contains that currency. If we don't filter it, we will end with an override for that machine without currencies.